edx/edx-app-ios
# Migrate to Firebase

### Description

[LEARNER-6784](https://openedx.atlassian.net/browse/LEARNER-6784)

I have migrated the Crashlytics from fabric to firebase, in this PR i updated the firebase core library and update the code which need to update in new version. We can also log our events on firebase by enabling the analytics from config file.

### Notes

### How to test this PR
We can take look into firebase the Crashlytics have started appear and properly migrated from fabric.
Firebase console 
https://console.firebase.google.com/u/0/project/openedx-mobile/crashlytics/app/ios:org.edx.mobile/issues?buildIds=94395918&state=all&time=last-seven-days&type=all

Fabric console
https://fabric.io/edx-inc/ios/apps/org.edx.mobile/issues?time=last-seven-days&event_type=crash&subFilter=device&state=open&build%5B0%5D=2.17.1
